source/index.html.md

```diff
@@ -750,3 +754,33 @@ This endpoint retrieves a specific account.
 | --------- | ------------------------------------------- |
 | CID       | The ID of the campaign containing accounts. |
 | ID        | The ID of the account.                      |
+
+# Conversions
+
+## Create
+
+```javascript
+window.ll_conversion("my awesome conversion");
+```
+
+If you use our loop.js script you can create conversion from Javascript. Conversions will be created for each matched account's campaign.
+
```

Comment:
conversions will be created for campaign, when they match for an account belonging to the campaign.